lib/ecto/adapters/sql.ex

```diff
@@ -124,6 +126,24 @@ defmodule Ecto.Adapters.SQL do
         :ok
       end
 
+      def do_get_sources(meta) do
```

Comment:
Because we will be logging too much information that will get in the way of reading the rest. I am already on the fence about this feature because the source exists in the query, so this is just a convenience. If we are going to log all sources, then it is best to just read the query.